### PR TITLE
Adds a toggleable notification sound to prayers for admins

### PR DESCRIPTION
**Pull request by Darkmight9**
## What Does This PR Do
Adds a notification sound to prayers(the chaplain's PDA message sound) and a new preference option to toggle it on and off.

## Why It's Good For The Game
Some admins like to answer prayers (for good or ill, heh) but they can be easily missed if things are busy. Adding a notification sound, which can be toggled off, will help admins notice prayers.

## Changelog
:cl:
add: Adds a prayer notification sound for admins who like to answer prayers and a new admin preference option to turn it on and off.
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/pray.dm [hunk 1]
```diff
@@ -35,6 +35,9 @@
 	for(var/client/X in GLOB.admins)
 		if(check_rights(R_EVENT,0,X.mob))
 			to_chat(X, msg)
+			if(X.prefs.sound & SOUND_PRAYERNOTIFY)
+				X << 'sound/items/PDA/ambicha4-short.ogg'
```
Comment (Ty-Omaha) [inline code/modules/admin/verbs/pray.dm]: Use SEND_SOUND instead

Comment (Darkmight9) [reply code/modules/admin/verbs/pray.dm]: which part do I plug that into?

Comment (Ty-Omaha) [reply code/modules/admin/verbs/pray.dm]: the part thats sending the sound, line 39

Comment (Darkmight9) [reply code/modules/admin/verbs/pray.dm]: so something like this? SEND_SOUND(X << 'sound/items/PDA/ambicha4-short.ogg')

Comment (Ty-Omaha) [reply code/modules/admin/verbs/pray.dm]: you should search the code for examples
https://github.com/ParadiseSS13/Paradise/blob/f8662f0833dd5aeaecb865be5ac9dcf200a4fe7f/code/controllers/subsystem/afk.dm#L65

Comment (Ty-Omaha) [reply code/modules/admin/verbs/pray.dm]: yes like that

**Comment by Crazylemon64** [comment]
What's meant by "SQL change" is that this causes a change in the SQL db, and can't simply be test-merged without permanent effects on the db

**Comment by AffectedArc07** [comment]
SQL works, and if @Kyep wants to automatically enable the prayer sound for everyone by default (Since its default in the code for new people), this query needs to be ran
```sql
UPDATE player SET sound=sound+1024;
```

**Comment by Fox-McCloud** [comment]
I kinda like how TG handles this; the chaplain gets preference (because he's speshul), but others don't.

**Comment by Fox-McCloud** [comment]
I'm not so keen on adding more sound toggles at this point in time, to be honest.

**Comment by Darkmight9** [comment]
I think it has use and merrit especially for admins that like to answer prayers

**Comment by AffectedArc07** [comment]
Once these conflicts are resolved, this should be able to be merged.

**Comment by Kyep** [comment]
@Darkmight9

**Comment by AffectedArc07** [comment]
Oh and if anyone is concerned that the `sound` column is running low on space, `sound` has the signed `MEDIUMINT` datatype.
image
It can fit that many toggles (24 bitflags). This PR is at 2048 (After deconfliction), which is flag 11. This PR is safe to merge.

**Comment by Darkmight9** [comment]
Roger I'll see about getting this deconflicted when I get off of work today

**Comment by AffectedArc07** [comment]
New update query
```sql
UPDATE player SET sound=sound+2047;
```

**Comment by Darkmight9** [comment]
I'm way out of practice with Byond mind explaining what that means/where that's supposed to go?

**Comment by AffectedArc07** [comment]
Its for @Kyep when he does the DB update for this to auto-enable it for everyone already

**Pull request merged**